microsoft/testsuites/performance/storageperf.py

```diff
@@ -224,7 +224,7 @@ def _perf_nfs(
             and not isinstance(server_node.os, Debian)
             and not isinstance(server_node.os, SLES)
         ):
-            raise SkippedException(f"{server_node.os} not supported")
+            raise SkippedException(f"{server_node.os.name} not supported")
```

Comment:
Can we change it to FailedException? We may review it later, if it should be supported.

Reply:
let's update in another pr after dicussion.